### DISCUSSION TITLE
Set budget cycle for weekly?

### DISCUSSION THREAD

**Discussion by deanfourie1** [firefly-iii/firefly-iii#8713] [Firefly III]
Hi is it possible to set the budgeting cycle to be weekly rather than monthly?

Thanks

**Comment by JC5** [comment]
Check out your preferences.

**Reply by deanfourie1** [reply]
Sorry, in the web app or the variables?

**Reply by JC5** [reply]
This is the kind of stuff that's in the documentation.

https://docs.firefly-iii.org/explanation/financial-concepts/budgets/
